src/Client/Components/ExpansionList/ExpansionListItem/ExpansionListItemContent/AppExpansionListItemContent.razor.cs

```diff
@@ -8,4 +8,19 @@ public partial class AppExpansionListItemContent
 {
     [Parameter]
     public InterviewQuestion? Question { get; set; }
+
+    [Parameter]
+    public EventCallback OnRatingChanged { get; set; }
+
+    public async Task RatingChanged(int rating)
+    {
+        if (Question is null)
+        {
+            throw new InvalidOperationException(nameof(Question));
```

Comment:
The first parameter to `InvalidoperationException` is a message, I'd go for something like this:

```suggestion
            throw new InvalidOperationException($"{nameof(Question)} cannot be null.");
```

to communicate the `null` a little better.